src/api/Metadata.js

```diff
@@ -257,6 +267,58 @@ class Metadata extends File {
         }
     }
 
+    /**
+     * Gets classification for a file.
+     *
+     * @param {BoxItem} file - File object for which we are getting classification
+     * @param {Function} successCallback - Success callback
+     * @param {Function} errorCallback - Error callback
+     * @param {boolean} forceFetch - True to clear the cache and make an api call
+     * @return {Promise}
+     */
+    async getClassification(
+        file: BoxItem,
+        successCallback: Function,
+        errorCallback: Function,
+        forceFetch: boolean = false,
+    ): Promise<void> {
+        const { id }: BoxItem = file;
+        if (!id) {
+            errorCallback(getBadItemError());
+            return;
+        }
+
+        const cache: APICache = this.getCache();
+        const key = this.getClassificationCacheKey(id);
+        this.successCallback = successCallback;
+        this.errorCallback = errorCallback;
+
+        // Clear the cache if needed
+        if (forceFetch) {
+            cache.unset(key);
```

Comment:
Even if we `forceFetch`, should we be clearing the cache entry? I'm thinking that in case the API errors out, at least we are in the same state as before versus having lost what we had already fetched

Reply:
It appears that ALL metadata calls clear out their cache if that flag is present. I think we want to to remain consistent. In regards to having what we had before, the calling component's state shouldn't be updated with new data, so in any case, old data will still be present in the UI

Reply:
There is another option, that is not yet in this method (can be added if needed) which is `refreshCache`, which is when you dont unset the cache but still fetch. Most api options take both forceFetch and refreshCache.

Reply:
IMO we should implement refreshCache here